### DISCUSSION TITLE
Sections on the page e.g. "about me"

### DISCUSSION THREAD

**Discussion by kszpieg** [uPagge/uBlogger#88] [Q&A] [answered]
Hello, @uPagge 

I'm curious how you did these cool-looking sections in your "About me" page. In the "post" archetype new markdown's header doesn't divide the background so... Can you tell me how to do this? Or you can share a template... if you can of course. :D 

![image](https://user-images.githubusercontent.com/38509982/116710785-6fde5480-a9d2-11eb-99cd-f7583909a830.png)

**Comment by uPagge** [comment, marked as answer]
Separations in articles can be done as follows

`{{% split-wide %}}{{% /split-wide %}}`

Inside the shortcode, you can put, for example, the title of the chapter heading

Then it will be like this

![изображение](https://user-images.githubusercontent.com/40397740/116818454-24fc4280-ab74-11eb-9052-95a7d796870d.png)

You can also place pictures and any other content.

**Reply by kszpieg** [reply]
Great, thanks for the help :D

**Reply by hellt** [reply]
@uPagge how can we change the selection of "share" links under the heading (as in your screenshot you have telegram/twitter/etc)
